### DISCUSSION TITLE
Ability to share portfolio

### DISCUSSION THREAD

**Discussion by BjornGrylls** [ghostfolio/ghostfolio#2131] [Q&A] [answered]
Hi
I might not be looking the right places, but I cannot seem to find a way to share my portfolio so my friends can view it.
If this is not yet possible, then this is a feature request.
An example: https://simpleportfolio.app/portfolios/zSG9MPkI/

**Comment by dtslvr** [comment, marked as answer]
Hi @BjornGrylls

You can share your portfolio from _My Ghostfolio_: _Grant access_ with an _Alias_, then you will get a sharable link.

**Reply by BjornGrylls** [reply]
Awesome. Thank you 
I didn't see the plus (+) in the bottom right of the screen.
